### DISCUSSION TITLE
How should I modify the onnx model, remove the argmax layer, and keep the original result

### DISCUSSION THREAD

**Discussion by ruisv** [open-mmlab/mmdeploy#1517] [Q&A] [answered]
<img width="999" alt="image" src="https://user-images.githubusercontent.com/27017322/206905454-319f80a7-7bef-4f38-8145-ab950bb6364b.png">

**Comment by lvhan028** [comment, marked as answer]
Hope https://github.com/onnx/onnx/blob/main/onnx/examples/Protobufs.ipynb helps

**Reply by ruisv** [reply]
Thanks
